SFBaseKitDemo/SFBaseKitDemo/ExampleLoginModule/LoginCoordinator.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  LogingCoordinator.swift
+//  SFBaseKitDemo
+//
+//  Created by Vesela Ilchevska on 15.04.20.
+//  Copyright © 2020 Upnetix. All rights reserved.
+//
+
+import Foundation
+import SFBaseKit
+
+protocol LoginSceneDelegate: class {
+    func sceneShouldContinueToForgottenPassword()
+    func sceneShouldContinueToLoginIn()
```

Comment:
Add the name of the scene in those methods.
If another scene also have continue to login method, the names will be the same - that is an issue for multi-protocol inheritance.
E.G. `loginSceneShouldContinueToLogIn()`

Reply:
Got that! Changed :)